src/jenkins_job_insight/monitoring.py

```
@@ -0,0 +1,493 @@
+# ---------------------------------------------------------------------------
+
+
+async def check_db(db_path: str) -> dict[str, str]:
+    """Check database connectivity."""
+    try:
+        import aiosqlite
+
+        async with aiosqlite.connect(db_path) as db:
+            await db.execute("SELECT 1")
+        return {"status": "ok"}
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)